Remove the CRD upgrade docs and link to older documentation instead to streamline the maintenance effort (#5186)
strimzi/strimzi-kafka-operator: documentation/assemblies/upgrading/assembly-upgrade-resources.adoc

[id='assembly-upgrade-resources-{context}']
= Strimzi custom resource upgrades

When upgrading Strimzi to {ProductVersion} from {ConvertAfterProductVersion} or earlier, you must ensure that your custom resources are using API version `{ApiVersion}`.

> You have to upgrade the Custom Resource Definitions and the Custom Resources before upgrading to Strimzi {ConvertBeforeProductVersion} or newer.

Reviewer: ```suggestion
You must upgrade the Custom Resource Definitions and the custom resources *before* upgrading to Strimzi {ConvertBeforeProductVersion} or newer.
```

> To do the upgrade, you can use the _API conversion tool_ from Strimzi {LastReleaseWithCRDUpgrade}.

Reviewer: ```suggestion
To perform the upgrade, you can use the _API conversion tool_ provided with Strimzi {LastReleaseWithCRDUpgrade}.
```
